po/pt_BR.po

```diff
@@ -70,189 +70,210 @@ msgstr "Angelo Verlain"
 #: data/com.vixalien.sticky.appdata.xml.in.in:54
 msgid "print where notes are logged when the app starts up (for syncing)"
 msgstr ""
+"imprime onde as notas são registradas quando o aplicativo é iniciado (para "
+"sincronização)"
 
 #: data/com.vixalien.sticky.appdata.xml.in.in:55
 msgid "add a flag `--new-note` to quicky jote down a note"
-msgstr ""
+msgstr "adicione um sinalizador '--new-note' para anotar rapidamente uma nota"
 
 #: data/com.vixalien.sticky.appdata.xml.in.in:62
 msgid "quickly open links in your notes"
-msgstr ""
+msgstr "abra links rapidamente nas suas notas"
 
 #: data/com.vixalien.sticky.appdata.xml.in.in:69
 msgid "changed GNOME Runtime version to 44"
-msgstr ""
+msgstr "alterado a versão do GNOME Runtime para 44"
 
 #: data/com.vixalien.sticky.appdata.xml.in.in:70
 msgid "updated dutch translation"
-msgstr ""
+msgstr "tradução holandesa atualizada"
 
 #: data/com.vixalien.sticky.appdata.xml.in.in:76
 msgid "fix packaging issues"
-msgstr ""
+msgstr "corrige problemas de empacotamento"
 
 #: data/com.vixalien.sticky.appdata.xml.in.in:81
 msgid "this release enhances usability and reliability"
-msgstr ""
+msgstr "esta versão melhora a usabilidade e confiabilidade"
 
 #: data/com.vixalien.sticky.appdata.xml.in.in:83
 msgid "updated the icon to make it more distinguished"
-msgstr ""
+msgstr "atualizado o ícone para torná-lo mais distinto"
 
 #: data/com.vixalien.sticky.appdata.xml.in.in:84
 msgid "updated translations (french, russian) and added a czech translation"
 msgstr ""
+"traduções atualizadas (francês, russa) e adicionado uma tradução em tcheco"
```

Comment:
It is totally fine to be picky. Thank you for that.

To keep consistency, should I maintain 'tradução tcheca' and 'tradução holandesa'? I got a bit confused on that.

Reply:
I'm just suggesting "czech translation", "dutch translation" and other "X translation" to be translated in the same way, and by that I mean using either "tradução em X" or "tradução Y" for all, instead of "tradução em tcheco" here and "tradução holandesa" there.